src/signals/incident-management/components/CheckboxList/CheckboxList.test.js

```diff
@@ -409,6 +409,9 @@ describe('signals/incident-management/components/CheckboxList', () => {
 
     expect(randomCheckbox.checked).toEqual(false);
 
+    const focusedCheckbox = container.querySelector('input[type="checkbox"]:focus');
```

Comment:
is this not to generic? How do I know the focused element is the right one?

Reply:
I've changed it to use `document.activeElement`